<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntitiesDialog</class>
 <widget class="QDialog" name="EntitiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="entitiesList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="privsList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="rolesList">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="privsInEntityList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="rolesInEntityList">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entities</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All LPrivs</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All Roles</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LPrivs in Entity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roles in Entity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="removePrivFromEntitiyButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/downarrow);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addPrivToEntityButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/uparrow);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="removeRoleFromEntityButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/downarrow);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addRoleToEntityButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/uparrow);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
